cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/utils/RandomUtils.java

```diff
@@ -100,8 +119,18 @@ public static InetAddress randomInet()
         return InetAddresses.fromInteger(RANDOM.nextInt());
     }
 
-    public static Object randomValue(CqlField.CqlType type)
+    public static String randomAlphanumeric(int minLengthInclusive, int maxLengthExclusive)
+    {
+        return randomAlphanumeric(RandomUtils.nextInt(minLengthInclusive, maxLengthExclusive));
+    }
+
+    public static String randomAlphanumeric(int maxLength)
     {
-        return type.randomValue(MIN_COLLECTION_SIZE);
+        StringBuilder sb = new StringBuilder(maxLength);
+        IntStream.rangeClosed(0, maxLength)
+                 .map(i -> RANDOM.nextInt(ALPHANUMERIC.length()))
+                 .mapToObj(ALPHANUMERIC::charAt)
+                 .forEach(sb::append);
+        return sb.toString();
```

Comment:
The alternative in my mind is random from [0, 36), and map the lower 10 to numbers and upper 26 to chars. Therefore, you do not need to look up the ALPHANUMERIC string. 
It is just a nit, feel free to ignore.